docs/jdbc-doris-catalog.md

```diff
@@ -41,16 +41,16 @@ more details.
-| `jdbc.pool.max-size`      | The maximum number of connections in the pool. `10` by default.                                                                                                     | `10`                                                    | No       | 0.5.0         |
-| `replication_allocation`  | The number of replications for the table. If not specified and the backend server less than 3, then Gravitino use will the default value `tag.location.default: 1`. | `tag.location.default: 1` if backend server less than 3 | No       | 0.6.0         |
+| Configuration item   | Description                                                                                                                           | Default value                     | Required | Since Version |
+|----------------------|---------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------|----------|---------------|
+| `jdbc-url`           | JDBC URL for connecting to the database. For example, `jdbc:mysql://localhost:9030`                                                   | (none)                            | Yes      | 0.5.0         |
+| `jdbc-driver`        | The driver of the JDBC connection. For example, `com.mysql.jdbc.Driver`.                                                              | (none)                            | Yes      | 0.5.0         |
+| `jdbc-user`          | The JDBC user name.                                                                                                                   | (none)                            | Yes      | 0.5.0         |
+| `jdbc-password`      | The JDBC password.                                                                                                                    | (none)                            | Yes      | 0.5.0         |
+| `jdbc.pool.min-size` | The minimum number of connections in the pool. `2` by default.                                                                        | `2`                               | No       | 0.5.0         |
+| `jdbc.pool.max-size` | The maximum number of connections in the pool. `10` by default.                                                                       | `10`                              | No       | 0.5.0         |
+| `jdbc.pool.max-size` | The maximum number of connections in the pool. `10` by default.                                                                       | `10`                              | No       | 0.5.0         |
+| `replication_num`    | The number of replications for the table. If not specified and the number of backend server less than 3, then the default value is 1. | `1` if backend server less than 3 | No       | 0.6.0         |
```

Comment:
what's the default value if not specified and the number of backend servers not less than 3?

Reply:
`1` if backend server less than 3

Reply:
I say what if not less than 3

Reply:
Number of copies. **The default number of copies is 3**. If the number of BE nodes is less than 3, you need to specify that the number of copies is less than or equal to the number of BE nodes.


For more, please see: https://doris.apache.org/docs/1.2/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-TABLE/

<img width="965" alt="image" src="https://github.com/user-attachments/assets/ff1a0ff1-5dfc-481e-9c42-43d326b80566">

Reply:
So, if the user does not specify the property value in Gravitino, you will use 1 by default when BE nodes are less than 3, otherwise use 3 by default, right?

Reply:
Yeah.

Reply:
plz document it